pkg/network/openshift_sdn.go

```diff
@@ -260,12 +261,23 @@ func fillOpenShiftSDNDefaults(conf, previous *operv1.NetworkSpec, hostMTU int) {
 	// If it's not supplied, we infer it from  the node on which we're running.
 	// However, this can never change, so we always prefer previous.
 	if sc.MTU == nil {
-		var mtu uint32 = uint32(hostMTU) - 50 // 50 byte VXLAN header
+		var mtu uint32
 		if previous != nil &&
 			previous.DefaultNetwork.Type == operv1.NetworkTypeOpenShiftSDN &&
 			previous.DefaultNetwork.OpenShiftSDNConfig != nil &&
 			previous.DefaultNetwork.OpenShiftSDNConfig.MTU != nil {
 			mtu = *previous.DefaultNetwork.OpenShiftSDNConfig.MTU
+		} else {
+			// utter paranoia
```

Comment:
We did, it was previously done in `FillDefaults`, this changed in this PR

Reply:
The logic that determines whether or not we should bother probing MTU is slightly complex, so I kept this in to ensure we don't accidentally break the CNO. Hopefully this never happens.